packages/conform-dom/form.ts

```diff
@@ -105,8 +105,8 @@ export type Constraint = {
 	required?: boolean;
 	minLength?: number;
 	maxLength?: number;
-	min?: string | number;
-	max?: string | number;
+	min?: string | number | Date;
+	max?: string | number | Date;
```

Comment:
not sure about this part. If it's safe to extend the type with `Date`. This was added to allow the developer to set `min` and `max` Dates for the timestamp field.